### PR TITLE
Update readme-Installation instructions

### LINKED ISSUE #336: Update Instructions for getting started
## what
* Better describe the process for getting started writing documentation. Add missing steps and dependencies.

image

## why
* It's not sufficient for someone to get started from zero

### PR DESCRIPTION
**Pull request by drnickiw**
## what
* Updated instructions for getting started with the documentation

## why
* Previous instructions were incomplete
* Closes #336

### CODE REVIEW

File: README.yaml [hunk 1]
```diff
@@ -28,9 +28,21 @@ include:
...
-  3. Run `make build` to generate static site in `public/`
+  1. Download [Docker](https://www.docker.com/).
+  2. Install the [Atom editor](https://atom.io/)(optional).
+
+    *After installation, run `make deps/atom`.
+
+    *Select _Activate the Atom_ --> _Install Shell Commands_ menu item.
+
+    *Ensure that /usr/local/bin is in the PATH.
+  3. Run the following commands for faster development:
```
Comment (osterman) [inline README.yaml]: 3. This is some text
    ```
    this is a code block
    /* with some code */
   ````
4. This is the next bullet

@drnickiw click edit on this comment to see how to write the markdown. Basically you need to use spaces (not tabs) to indent the code block. Then it should work.

File: README.yaml [hunk 2]
```diff
@@ -28,9 +28,21 @@ include:
   - "docs/targets.md"
 
 usage: |-
-  1. Run `make init`
-  2. Run `COMPONENTS_BUILD=true/false make run` to start local server
-  3. Run `make build` to generate static site in `public/`
+  1. Download [Docker](https://www.docker.com/).
+  2. Install the [Atom editor](https://atom.io/)(optional).
+
+    *After installation, run `make deps/atom`.
```
Comment (osterman) [inline README.yaml]:
```suggestion
     * After installation, run `make deps/atom`.
```

File: README.yaml [hunk 3]
```diff
@@ -28,9 +28,21 @@ include:
...
+
+    *Select _Activate the Atom_ --> _Install Shell Commands_ menu item.
```
Comment (osterman) [inline README.yaml]:
```suggestion
     * Select _Activate the Atom_ --> _Install Shell Commands_ menu item.
```

File: README.yaml [hunk 4]
```diff
@@ -28,9 +28,21 @@ include:
...
+
+    *Ensure that /usr/local/bin is in the PATH.
+  3. Run the following commands for faster development:
+```export YARN_BUILD_DISABLED=true
```
Comment (osterman) [inline README.yaml]: Indent the code block

File: README.yaml [hunk 5]
```diff
@@ -28,9 +28,21 @@ include:
...
+export UTTERANCES_BUILD_DISABLED=true```
+  4. Run `make init`.
+  5. Run `COMPONENTS_BUILD=true/false make run` to start the local server.
```
Comment (osterman) [inline README.yaml]: This is correct shell notation for passing the environment variables, however, personal preference for `make` is to put them at the end of the line b/c it looks cleaner. `make` supports this convention.
```suggestion
  5. Run `make run COMPONENTS_BUILD=true` to start the local server. Specify `false` to disable components building.
```

File: README.yaml [hunk 6]
```diff
@@ -28,9 +28,23 @@ include:
...
 usage: |-
-  1. Run `make init`
-  2. Run `COMPONENTS_BUILD=true/false make run` to start local server
-  3. Run `make build` to generate static site in `public/`
+  1. Download [Docker](https://www.docker.com/).
+  2. Install the [Atom editor](https://atom.io/)(optional).
+
+    * After installation, run `make deps/atom`.
+
+    * Select _Activate the Atom_ &#8594; _Install Shell Commands_ menu item.
```
Comment (osterman) [inline README.yaml]: You can use the html entity, but let's just use thee raw utf8 character (→), since that's supported by modern browsers.

Comment (osterman) [inline README.yaml]:
```suggestion
    * Select _Activate the Atom_ →  _Install Shell Commands_ menu item.
```

File: content/documentation/standards.md [hunk 7]
```diff
@@ -0,0 +1,6 @@
+## Documentation Standards
```
Comment (osterman) [inline content/documentation/standards.md]: Did you mean to commit this file with this PR?

Comment (drnickiw) [reply content/documentation/standards.md]: @osterman  no i didn't. This was part of my commit issues from before.

File: README.yaml (hunk 6 above)
Comment (drnickiw) [reply README.yaml]: Ok I'll update this.

File: content/documentation/standards.md (hunk 7 above)
Comment (osterman) [reply content/documentation/standards.md]: You can just delete it from your commit and push.

File: README.yaml [hunk 8]
```diff
@@ -28,9 +28,21 @@ include:
   - "docs/targets.md"
 
 usage: |-
-  1. Run `make init`
-  2. Run `COMPONENTS_BUILD=true/false make run` to start local server
-  3. Run `make build` to generate static site in `public/`
+  1. Download [Docker](https://www.docker.com/).
+  2. Install the [Atom editor](https://atom.io/)(optional).
+
```
Comment (osterman) [inline README.yaml]: Try playing with the spacing here. The formatting is still wrong. Bullets should be indented.

image

**Comment by osterman** [comment]
This file is still in the commit.

image

**Pull request merged**